compiler/damlc/tests/daml-test-files/Set.daml

```diff
@@ -1,11 +1,6 @@
 -- Copyright (c) 2020, Digital Asset (Switzerland) GmbH and/or its affiliates.
 -- All rights reserved.
 
--- @INFO range=57:30-57:77; Use infix
--- @INFO range=58:21-58:52; Use infix
--- @INFO range=59:24-59:58; Use infix
--- @INFO range=60:17-60:54; Use infix
```

Comment:
I'm not convinced this is a great hint. I'm wondering if we should rather steer away from using infix syntax in our examples instead of pushing our users in that direction. Any opinions, @cocreature @bame-da?

Reply:
We already disable it in `create-daml-app` so I would be open to disabling it by default. However, I have to admit that I do actuallylike this hint for things like `elem` where I find the infix code easier to read. Contrary to some of the other hints like `DA.Bifunctor` I’m also not sure that you can really avoid stumbling upon infix notation so the costs of introducing unfamiliar syntax past the very beginning in `create-daml-app` are relatively low.